test/controllers/changeGroupLead/MainContactJobTitleControllerSpec.scala

```diff
@@ -21,30 +21,29 @@ import controllers.BetterMockActionSyntax
 import controllers.actions.JourneyAction
 import controllers.actions.JourneyAction.{RequestAsyncFunction, RequestFunction}
 import forms.changeGroupLead.MainContactJobTitleFormProvider
+import models.Mode.NormalMode
```

Comment:
i think your inteliJ is getting a bit agressive with import optimisation which is making PR's cloudy